packages/wrangler/src/worker.ts

```diff
@@ -157,6 +163,7 @@ export interface CfWorkerInit {
     data_blobs: CfDataBlobBindings | undefined;
     durable_objects: { bindings: CfDurableObject[] } | undefined;
     r2_buckets: CfR2Bucket[] | undefined;
+    services: CfService[] | undefined;
```

Comment:
If there is a default empty array, then undefined would no be necessary 
```suggestion
    services: CfService[] | [];
```
Then the services length check would also not need a coercion. Alternatively it could be checked for `undefined` before that length check and set to empty array.

Reply:
I'm not sold on whether the default should an array (including for some other fields) so I'm staying defensive here.

Reply:
That our validation automatically sets it so makes me uncomfortable, and I want to revisit it sometime.

Reply:
> I want to revisit it sometime.

Fair enough then 😄